postcss-selector-parser.d.ts

```diff
@@ -311,6 +351,51 @@ declare namespace parser {
          * The attribute name after having been qualified with a namespace.
          */
         readonly qualifiedAttribute: string;
+
+        /**
+         * The case insensitivity flag or an empty string depending on whether this
+         * attribute is case insensitive.
+         */
+        readonly insensitiveFlag : 'i' | '';
```

Comment:
Not sure if we care, (or if this is handled in code and normalized) but the case insensitive flag is **technically**... Insensitive :) It can be `i` or `I`.

Reply:
This was a good thing to note. the property always returns lowercase, but there was some other small issues around using a capital `I`.